subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1219,17 +1101,6 @@ static int prop_set(struct net_buf_simple *buf,
 	case BT_MESH_LIGHT_CTRL_PROP_ILLUMINANCE_STANDBY:
 		srv->reg.cfg.lux[LIGHT_CTRL_STATE_STANDBY] = val;
 		break;
-#else
-	case BT_MESH_LIGHT_CTRL_COEFF_KID:
-	case BT_MESH_LIGHT_CTRL_COEFF_KIU:
-	case BT_MESH_LIGHT_CTRL_COEFF_KPD:
-	case BT_MESH_LIGHT_CTRL_COEFF_KPU:
-	case BT_MESH_LIGHT_CTRL_PROP_REG_ACCURACY:
-	case BT_MESH_LIGHT_CTRL_PROP_ILLUMINANCE_ON:
-	case BT_MESH_LIGHT_CTRL_PROP_ILLUMINANCE_PROLONG:
-	case BT_MESH_LIGHT_CTRL_PROP_ILLUMINANCE_STANDBY:
-		break; /* Prevent returning -ENOENT */
-#endif
```

Comment:
This highlights the need for either creating an additional kconfig flag for "default regulator" or restructuring the connection with the regulator, IMO. `CONFIG_BT_MESH_LIGHT_CTRL_SRV_REG` should either clearly disable all regulator business (custom or standard), or just remove the standard regulator and all its state. At the moment, it's a mix of both.